src/bp-core/compatibility/bp-incompatible-plugins-helper.php

```diff
@@ -527,3 +527,55 @@ function bp_core_set_uri_elementor_show_on_front( $bool ) {
 	return $bool;
 }
 add_filter( 'bp_core_set_uri_show_on_front', 'bp_core_set_uri_elementor_show_on_front', 10, 3 );
+
+/**
+ * Prevent BB template redering and Redirect to the Elementor "Maintenance Mode" template.
+ *
+ * @since BuddyBoss 1.5.8
+ *
+ * @return void
+ */
+function bp_prevent_template_rendering() {
```

Comment:
change function name to `bb_get_elementor_maintenance_mode_template`

Reply:
Update the function bp_prevent_template_rendering to bb_get_elementor_maintenance_mode_template.
https://github.com/buddyboss/buddyboss-platform/commit/b047fd6f00ea2b18a321e9d621667dad7e5af2a6